### PR TITLE
Improve downgrade / CIDR restrictions documentation

### PR DESCRIPTION
**Pull request by joestringer**
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/5186)

### CODE REVIEW

**Comment by joestringer** [comment]
test-docs-please

File: Documentation/install/upgrade.rst [hunk 1]
```diff
@@ -420,8 +420,98 @@ also disable the usage of the feature.
...
+        result: 32, max: 18)
+
+The supported count of unique prefix lengths may differ between Cilium minor
+releases, for example Cilium 1.1 supports 20 unique prefix lengths on Linux
+4.10 or older, while Cilium 1.2 only supports 18 (for IPv4) or 4 (for IPv6).
+
+Mitigation
+^^^^^^^^^^
+
+Users may construct CIDR policies that use less unique prefix lengths, by
```
Comment (ianvernon) [inline Documentation/install/upgrade.rst]: , by --> , by, for example,

**Review by ianvernon — APPROVED** [review]
One small nit. Well-written and easy to understand!

File: Documentation/install/upgrade.rst (hunk 1 above)
Comment (joestringer) [reply Documentation/install/upgrade.rst]: also, there's fewer of them, not less :-)

**Comment by joestringer** [comment]
test-me-please

**Pull request merged**